bluemira/power_cycle/tools.py

```diff
@@ -31,3 +37,65 @@ def create_axes(ax=None):
     if ax is None:
         _, ax = plt.subplots()
     return ax
+
+
+@nb.jit
+def unique_domain(x: np.ndarray, epsilon: np.number = DEFAULT_EPSILON):
```

Comment:
the default can just sit in the function definition in both cases, no need for a global variable. Also, the typing should just be float.
```suggestion
def unique_domain(x: np.ndarray, epsilon: float = 1e-10):
```